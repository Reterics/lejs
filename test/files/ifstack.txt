{#test}}
something2
{#test2}}
something
{#test3}}
Test3 is true
{/#test3}}
{#!test3}}
Test3 is false
{/#test3}}
{/#test2}}
something3
{/#test}}