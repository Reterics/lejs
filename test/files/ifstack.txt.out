
something2

something


Test3 is false


something3
